src/flow/flowcomm.js

```diff
@@ -449,6 +451,7 @@ const vowToInner = new WeakMap();
 const resolverToInner = new WeakMap();
 
 // TODO change to throw TypeError if these aren't present.
+/* eslint-disable-next-line no-unused-vars */
```

Comment:
do we know why it's complaining here? `value` is used on the very next line.

Reply:
it's actually `validInnerResolver` that is never used!